Gemfile.lock

```diff
@@ -155,6 +155,9 @@ DEPENDENCIES
   sdoc (~> 0.4.0)
   spring
   sqlite3
+<<<<<<< HEAD
+=======
   thin
+>>>>>>> upstream/master
```

Comment:
Hmm, this shows three lines added that shouldn't be in there -- can you remove those lines?